container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -276,6 +276,16 @@ if [ ! -f ${TEMPEST_PATH}exclude.txt ] && [ -z ${TEMPEST_EXCLUDE_LIST} ]; then
     touch ${TEMPEST_PATH}exclude.txt
 fi
 
+# This workaround is required for the whitebox-neutron-tempest plugin. We need
+# to be able to specify 600 permissions for the id_ecdsa.
+if [ -f ${HOMEDIR}/id_ecdsa ]; then
```

Comment:
Yes, the previous behavior of the test-operator was to mount the key under `/var/lib/tempest/.ssh` but this will be changed with these two PRs:

- https://github.com/openstack-k8s-operators/tcib/pull/159
- https://github.com/openstack-k8s-operators/test-operator/pull/69

Reply:
I guess that from the previous comments it can be a little bit confusing but it is actually the other way around. [The patch in the test-operator](https://github.com/openstack-k8s-operators/test-operator/pull/69/files) depends on this change. 

Merging this change before the one in the test-operator should not have any influence on the run_tempest.sh thanks to this condition:
```
if [ -f ${HOMEDIR}/id_ecdsa ]; then
```